CIScripts/Testenv/Testbed.Impl.ps1

```diff
@@ -0,0 +1,73 @@
+class Testbed {
+    [string] $Name
+    [string] $VMName
```

Comment:
What is the difference between these two variables?

Reply:
The second one is used when You have the setup on HyperV and connect to them not by network but some HyperV names.